### PR TITLE
BOOKKEEPER-1081: a docker file for Apache BookKeeper

### PR DESCRIPTION
**Pull request by jiazhai**
A docker file for Apache BookKeeper.  This docker image will start a new bookie instance.
Steps to run it:
1, start zookeeper service;
2, run this docker image, e.g:
```
$ docker run -t -i -e "ZK_URL=192.168.100.100:2181" bookkeeper/bookie /bin/bash
```

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM java:openjdk-8-jre-alpine
```
Comment (eolivelli) [inline docker/Dockerfile]: Is it working with alpine?
On alpine there are issues with netty native

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# docker-distributedlog
```
Comment (eolivelli) [inline docker/README.md]: Typo from cut and paste from dl?

Comment (jiazhai) [inline docker/README.md]: Thanks. will change it.

File: docker/Dockerfile (hunk 1 above)
Comment (jiazhai) [inline docker/Dockerfile]: How to verify this? There seems no error reported when I start 3 bookies, and did some simple write.

**Comment by jiazhai** [comment]
Since there is a dup related work, will close this PR first, discuss and collaborate together with @eolivelli and @caiok to make it done.

**Pull request closed without merging**